Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Utilities/Language/LanguageManagerOld.cs

```diff
@@ -23,19 +23,19 @@ namespace Krypton.Toolkit.Suite.Extended.Outlook.Grid
     /// </summary>
     // Note: Overhaul this!
     public class LanguageManager
-    {        
-        private static LanguageManager _mInstance = null;
+    {
+        private static LanguageManager? _mInstance = null;
 
-        private static readonly object _mylock = new object();
+        private static readonly object Mylock = new();
```

Comment:
it's a private (I know resharper has done this, so change the rule) so needs to be lower case

Reply:
Done